web/themes/new_weather_theme/templates/partials/daily-forecast-list-item.html.twig

```diff
@@ -68,21 +68,24 @@ only be a nighttime period.
       </div>
       {% endfor %}
     </div>
-    <wx-hourly-toggle class="text-primary margin-top-1" role="button" aria-expanded="false" aria-controls="{{itemId}}-hourly-table" data-expanded-text="{{ 'Hide hourly details' | t}}" data-hidden-text="{{'Show hourly details' | t}}" data-expanded-icon-name="expand_less" data-hidden-icon-name="expand_more" tabindex="0">
+    <wx-hourly-toggle class="text-primary-dark margin-top-1" role="button" aria-expanded="false" aria-controls="{{itemId}}-hourly-togglee" data-expanded-text="{{ 'Hide hourly details' | t}}" data-hidden-text="{{'Show hourly details' | t}}" data-expanded-icon-name="expand_less" data-hidden-icon-name="expand_more" tabindex="0">
```

Comment:
This is twice now I've seen `togglee` here and thought it was a typo. But nope. 😂

Reply:
Maybe we change to `toggle-target`?

Reply:
Naw, I think it's fine. I laughed both times.